dto/src/test/java/ca/on/oicr/gsi/runscanner/rs/dto/test/AbstractNotificationDtoTest.java

```diff
@@ -33,10 +32,7 @@ public void testPartiallyPopulatedNotificationRoundTrip() throws Exception {
     notificationDto.setHealthType(HealthType.RUNNING);
 
     ObjectMapper mapper = new ObjectMapper();
-    mapper
-        .registerModule(new JavaTimeModule())
-        .setDateFormat(new ISO8601DateFormat())
-        .enable(SerializationFeature.INDENT_OUTPUT);
+    mapper.registerModule(new JavaTimeModule()).enable(SerializationFeature.INDENT_OUTPUT);
```

Comment:
The ISO8601DateFormat class was marked as deprecated and also doesn't match the code it's testing since those two lines were the only usages of ISO8601DateFormat in the codebase.